fcrepo-clients/src/test/java/edu/unc/lib/dl/event/RepositoryPremisLoggerIT.java

```diff
@@ -81,7 +93,10 @@ public void init() throws Exception {
                     @Override
                     public URI answer(InvocationOnMock invocation) throws Throwable {
                         InputStream contentStream = invocation.getArgumentAt(1, InputStream.class);
-                        copyInputStreamToFile(contentStream, path.toFile());
+                        Path tempFilePath = createTempFile("temp_content", null);
+                        copyInputStreamToFile(contentStream, tempFilePath.toFile());
+                        contentStream.close();
```

Comment:
Its not doing any harm, but you don't need to close the stream in this case. Most InputStream read operations close the stream, and this particular one specifies that it does so https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#copyInputStreamToFile(java.io.InputStream,%20java.io.File)